optimizely/event/event_payload.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2019 Optimizely
```

Comment:
The word event in `event_payload` is redundant as it is already under event package. Perhaps just say `payload.py`